### PR TITLE
Cronjobs done

### PR DESCRIPTION
**Pull request by OzanKurt**

### CODE REVIEW

File: includes/common.php [hunk 1]
```diff
@@ -69,8 +78,11 @@ function stripslashes_gpc(&$value)
 ini_set('error_log', 'includes/error.log');
 
 require 'includes/GeneralFunctions.php';
-set_exception_handler('exceptionHandler');
-set_error_handler('errorHandler');
+
+if (!isCommandLineInterface()) {
+	set_exception_handler('exceptionHandler');
+	set_error_handler('errorHandler');
+}
```
Comment (jkroepke) [inline includes/common.php]: We should add set_exception_handler for cli-mode otherwise you will be get unreadable error message :-)

Comment (OzanKurt) [inline includes/common.php]: Didn't know that :)

Comment (jkroepke) [inline includes/common.php]: You can still push commits to your branch. It will be popup here.

Comment (OzanKurt) [inline includes/common.php]: Okkay, I will do the changes and commit ;)

**Comment by jkroepke** [comment]
Hopefully it will not break the crons! ;)

**Comment by OzanKurt** [comment]
:blush:

**Pull request merged**